docs/install/first-run.md

```diff
@@ -5,9 +5,13 @@ through the initial setup.
 
 By default, you can access the platform at [http://localhost:3000](http://localhost:3000).
 
+## 1. Begin
+Open FlowForge in your browser [http://localhost:3000](http://localhost:3000).  
+Click the **START SETUP** button
+
 <img src="images/setup-01.png" width=500 />
 
-## 1. Create an Administrator
+## 2. Create an Administrator
```

Comment:
We should just remove these numbers. That's much more stable

Reply:
There are references back to Step x

Will not be doing this now (but I am not opposed if you want to pick it up personally ;) )